src/app/clusters/group-key-mgmt-server/group-key-mgmt-cluster.cpp

```diff
@@ -1,6 +1,6 @@
 /**
  *
- *    Copyright (c) 2020-2022 Project CHIP Authors
+ *    Copyright (c) 2025 Project CHIP Authors
```

Comment:
since this file was modified from the original, probably:

```suggestion
 *    Copyright (c) 2020-2025 Project CHIP Authors
```